src/app/app.module.ts

```diff
@@ -1,23 +1,26 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { NgModule } from '@angular/core';
-import { HttpClientModule } from '@angular/common/http';
 
 import { SharedModule } from './shared/shared.module';
 import { AppRoutingModule } from './app-routing.module';
 import { CoreModule } from './core/core.module';
-import { ReduxModule } from './redux/redux.module';
+import { ReduxModule } from './store/redux.module';
```

Comment:
You might want to call this store module to match the directory name.

Reply:
Oops.  Will fix, before merging.